pythonforandroid/recipes/android/__init__.py

```diff
@@ -40,7 +40,7 @@ def prebuild_arch(self, arch):
 
         is_sdl2 = bootstrap_name in ('sdl2', 'sdl2python3', 'sdl2_gradle')
         is_pygame = bootstrap_name in ('pygame',)
-        is_webview = bootstrap_name in ('webview',)
+        is_webview = bootstrap_name in ('webview', 'service_only',)
```

Comment:
shouldn't there rather be a "is_serviceonly" added? right now the way it reads doesn't make much sense, which will confuse future coders fiddling with this code

Reply:
@JonasT:

thanks for the hint, i have now added an 'is_service_only' flag as proposed, so the PR should be ready for merge